modules/virt-about-efi-mode-for-vms.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt-efi-mode-for-vms.adoc
+
+[id="virt-about-efi-mode-for-vms_{context}"]
+= About EFI mode for virtual machines
+
+`EFI` (Extensible Firmware Interface) is a firmware interface that initializes the hardware components and starts the operating system stored on a hard disk drive when a computer starts.
```

Comment:
I would suggest rewriting this sentence for more clarity and user focus, because I don't quite understand what you are trying to say. It might also be better to split it into two sentences. 

Specifically, I'm confused about what "the operating system" means. Is it an OS that is stored on a hard drive versus a different OS? 

s/stored on/that is stored on

Reply:
I would suggest rewriting the module so that you start by explaining why you might want to use EFI (faster boot), and *then* include the more technical information.